pyartcd/pyartcd/pipelines/build_microshift.py

```diff
@@ -267,28 +268,20 @@ async def _change_advisory_status(self):
         await exectools.cmd_assert_async(cmd, env=self._elliott_env_vars)
 
     @staticmethod
-    async def parse_release_payloads(payloads: Iterable[str]):
+    def parse_release_payloads(payloads: Iterable[str]):
         result = {}
-        pullspecs = []
         for payload in payloads:
-            if "/" not in payload:
-                # Convert nightly name to pullspec
-                # 4.12.0-0.nightly-2022-10-25-210451 ==> registry.ci.openshift.org/ocp/release:4.12.0-0.nightly-2022-10-25-210451
-                _, brew_cpu_arch, _ = isolate_nightly_name_components(payload)
-                pullspecs.append(constants.NIGHTLY_PAYLOAD_REPOS[brew_cpu_arch] + ":" + payload)
+            if "arm64" in payload:
+                brew_arch = "aarch64"
+                res = requests.get(f"{constants.RELEASE_CONTROLLER_ENDPOINT[brew_arch]}/{payload.split('-')[0]}-0.nightly-arm64/release/{payload}")
```

Comment:
This assumes a certain pullspec format like `4.12.0-0.nightly-2022-10-25-210451`. This is probably not a big issue for automated stream build but we will certainly lose the flexibility to rebase onto any other kinds of payloads (like named releases or sha256sum).